esp/mlrs-wifi-bridge/mlrs-wifi-bridge.ino

```diff
@@ -65,7 +66,7 @@ int port_udp = 14550; // connect to this port per UDP // MissionPlanner default
 int baudrate = 57600;
 
 // WiFi channel
-int wifi_channel = 13; // 1 is the default, 13 (2461-2483 MHz) has the least overlap with mLRS 2.4 GHz frequencies
+int wifi_channel = 13; // 1 is the default, 13 (2461-2483 MHz) has the least overlap with mLRS 2.4 GHz frequencies, but is not used in the US
```

Comment:
maybe "... frequencies; also is not used in the US"

Reply:
The but is important.  Wifi Channel 13 is generally not available in the US.  It only goes to 11 :)

How about ... frequencies;  Note, channels higher than 11 may not work with US WiFi Clients.

Reply:
ah, it's probably a language issue on my side but the but still reads odd, it triggers a wrong thought. I find your explanation better. So, maybe_ "... frequencies. Note: Channel 13 is generally not available in the US, where 11 is max." ?
Since the line may become long you simply could put the whole comment inbetween // Wifi ... and int wifif_...